test/scripts/e2e-kind.sh

```diff
@@ -32,9 +32,6 @@ Services.+session affinity
 # TO BE IMPLEMENTED: https://github.com/ovn-org/ovn-kubernetes/issues/1116
 EndpointSlices
 
-# TO BE IMPLEMENTED: https://github.com/ovn-org/ovn-kubernetes/issues/1664
-should be able to preserve UDP traffic when server pod cycles for a NodePort service
```

Comment:
Why is this in this PR?

Reply:
because this e2e test verifies conntrack and nodeports, that are very influenced by this PR. 
I had a separate PR to enable it, I need to reopen it

Reply:
It seems like even if this PR fixes that test, it would only be fixing it when the new controller was enabled, and so we can't unskip the test unconditionally?

Reply:
we must skip it unconditionally, it is unskipped in openshift and is orthogonal to this PR, just that I wanted to test this e2e tets in this PR with the CI
It is my fault that is not enabled, I had a PR to enable it a few weeks ago and I closed it accidentally, this is the new PR https://github.com/ovn-org/ovn-kubernetes/pull/1889